libbeat/scripts/kibana_export.py

```diff
@@ -0,0 +1,47 @@
+from elasticsearch import Elasticsearch
+import argparse
+import os
+import json
+
+
+def dump_objects(es, output_directory, doc_type):
+    res = es.search(
+        index='.kibana',
+        doc_type=doc_type,
+        size=1000)
+
+    dir = os.path.join(output_directory, doc_type)
+    if not os.path.exists(dir):
+        os.makedirs(dir)
+
+    for doc in res['hits']['hits']:
+
+        # Fixes windows problem with files with * inside
+        # Removes it from index pattern
+        if doc_type == "index-pattern":
+            doc['_id'] = doc['_id'][:-2]
+
+        filepath = os.path.join(dir, doc['_id'] + '.json')
+        with open(filepath, 'w') as f:
+            json.dump(doc['_source'], f, indent=2)
+            print("Written {}".format(filepath))
+
+
+def main():
+    parser = argparse.ArgumentParser(
+        description="Dumps Kibana dashboards, vizualization and " +
+                    "searches in json files")
+    parser.add_argument("--url", help="Elasticsearch URL. E.g. " +
+                                      "http://localhost:9200.", required=True)
+    parser.add_argument("--dir", help="Output directory", default="saved")
```

Comment:
I was thinking it's useful in case someone is using Shield, but I agree that we can leave them out for now and add them on request.